Osiris/Hacks/Misc.h

```diff
@@ -13,6 +13,9 @@
 
 namespace Misc
 {
+    inline int teamDamage = 0;
```

Comment:
Should it really be in header? Maybe move into `Misc::teamDamageCounter` as a static variable.

Reply:
How would I convert it to a static variable? If I do then it is no longer changable in other functions. 
Would I make a function to reset it and call it instead of just setting them to zero?